### DISCUSSION TITLE
I suggest adding the ability to place a watering can in the weapon holder.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14426] [Ideas and suggestions]
This is convenient when the watering can is always in sight.
The watering can is quite suitable in size, so I was surprised when I could not fit the watering can in the weapon holder. Lets add this feature.

**Comment by jigardi** [comment]
Heya I really like this suggestion.
So I made a standalone mod for it: https://steamcommunity.com/sharedfiles/filedetails/?id=3444378756

The functionality is as requested, but I could not figure out how to dynamically alter the itemrotation attribute of the weaponholder, so that the watering can faces the correct way(upward).  I believe it is possible to add this via a StatusEffect, but it is currently beyond my skill.
